### PR TITLE
This should make typescript match javascript

### PR DESCRIPTION
**Pull request by DavidRGriswold**

### CODE REVIEW

File: themes/braver-solarized-dark-color-theme.json [hunk 1]
```diff
@@ -824,19 +824,19 @@
       }
     },
     {
-      "name": "JavaScript Variables",
+      "name": "JS/TS Variables",
```
Comment (braver) [inline themes/braver-solarized-dark-color-theme.json]: Perhaps move all these up to be near the earlier declarations for variables?

**Review by braver — COMMENTED** [review]
In general very much in agreement with this 👍🏻

**Comment by DavidRGriswold** [comment]
Okay, I pushed an update to do that. I also renamed them to not include "JS/TS" in the title, since they are now more generic and could potentially apply to other languages. I did leave a comment above the three moved element stating they will apply to javascript and typescript specifically, in case anybody is reading the code trying to figure out how to customize those elements.

**Comment by DavidRGriswold** [comment]
Oof, the git diff is doing a TERRIBLE job with those files. Didn't detect that it was just some inserted and deleted lines, not a complete rewrite! Look for the lines it highlights as syntax errors (comments not officially part of JSON, but work fine for vscode files) to see where the code starts.

**Comment by braver** [comment]
Weird. Ah well, whatever. Looks good 👍🏻

**Comment by DavidRGriswold** [comment]
All right I will merge it in in a few. 2.0.1 feels right or is this 2.1.0?

**Comment by DavidRGriswold** [comment]
Went ahead and merged. If you want to update the version before uploading to the microsoft place, feel free.

**Comment by braver** [comment]
Sure, but won’t be until next weekend though

**Comment by braver** [comment]
Published as 2.0.2!

**Pull request merged**